sunpy/visualization/animator/wcs.py

```diff
@@ -203,6 +203,7 @@ def plot_start_image(self, ax):
         elif self.plot_dimensionality == 2:
             artist = self.plot_start_image_2d(ax)
 
+        self.axes.reset_wcs(wcs=self.wcs, slices=self.slices_wcsaxes)
```

Comment:
Do we need to do this? `plot_start_image` should be called on initialisation just after a fresh axes has been created so as long as `self.wcs` and `self.slices_wcsaxes` haven't been modified between construction and this method getting called we don't need this.

The `reset_wcs` method is the one which does all the setup for `WCSAxes` so it's really expensive.

Reply:
I think this does some stuff that `plot_start_image` doesn't currently do. I thought this would fix some of the image failures, but it doesn't, so I might put this change in another draft PR to see what happens and perhaps fix `plot_start_image`.

Reply:
re. performance, `reset_wcs` is called **every time** the slider is changed, so unless you're plotting and saving the figure without changing the slider I'm not sure performance is too much of an issue.

Reply:
Running through the `reset_wcs` twice on startup if it's redundant seems like a waste, but if it makes writing a `plot_start_image` method easier then it is probably worth it.